analysis_notebooks/Geography Analysis.ipynb

```diff
@@ -306,154 +311,253 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "counts = pickle.load(open(\"results/{}/geo_ctr.pkl\".format(folder_name), \"rb\"))\n",
-    "iso3_to_subregion = pickle.load(open('util_files/iso3_to_subregion_mappings.pkl', 'rb'))\n",
-    "gc = GeonamesCache()\n",
-    "iso3_codes = list(gc.get_dataset_by_key(gc.get_countries(), 'iso3').keys())\n",
-    "\n",
-    "# https://ramiro.org/notebook/basemap-choropleth/\n",
-    "cm = plt.get_cmap('Blues')\n",
-    "bins = np.logspace(min(list(counts.values())), np.log2(max(list(counts.values()))+1), base=2.0)\n",
-    "num_colors = len(bins)\n",
-    "scheme = [cm(i / num_colors) for i in range(num_colors)]\n",
+    "# check if we use country counts method or not:\n",
```

Comment:
maybe have a more clear name than "country counts method or not". e.g., "Region label" vs "GPS label", or something like that